src/components/ClickCopy.vue

```diff
@@ -5,16 +5,16 @@
       <button
         class="text-sm font-medium focus:outline-none focus:shadow-outline text-app-text"
         type="button"
-        v-clipboard:copy="gradient"
+        v-clipboard:copy="value"
         v-clipboard:success="onCopy"
         v-clipboard:error="onError"
         :data-gradient="name"
       >
-        Copy CSS
+        Copy <span v-if="vanilla">Vanilla</span> CSS
       </button>
       <span
         class="inline-block w-6 h-1 mt-1 duration-100 ease-in-out rounded-full transition-size group-hover:w-full"
-        :class="gradient"
+        :class="gradient" ref="gradientUnderline"
```

Comment:
I would use the main "gradient" part as the `ref`.

Reply:
What do you mean by "main gradient part"? I don't understand the suggestion.

Reply:
Sorry, that was very vague 😅

[This line here](https://github.com/jordihales/hypercolor/blob/master/src/components/Gradient.vue#L3)